armi/materials/material.py

```diff
@@ -690,7 +690,71 @@ def density3(self, Tk=None, Tc=None):
         return self.density(Tk=Tk, Tc=Tc)
 
 
-class FuelMaterial(Material):
+class SimpleSolid(Material):
+    """
+    Base material for a simple material that primarily defines density
+
+    Notes
+    -----
+    This function assumed the density is defined on the _density method and
+    this base class keeps density, density3 and linearExpansion all in sync
+
+    class SimpleMaterial(SimpleSolid):
+
+        def _density(self, Tk=None, Tc=None):
+            "
+            density that preserves mass when thermally expanded in 3D.
+            "
+            ...
```

Comment:
@sombrereau Can you clarify what is meant here with the `_density()` method?

Based on the docstring, it appears that it should be necessary to define a `_density()` method, but looking around the code, I don't ever see one.

Am I reading this incorrectly? Or is this docstring out of date?

Reply:
Out of date.

I am working on a PR that moves:

* `Material.density()` to `Material.pseudoDensity()`
* `Material.density3()` to `Material.density()`

And in that branch, I have already fixed this docstring.

(Also, spoilers, I found a bunch of bugs while making that branch.  Joy!)